.github/workflows/phpstan.yml

```diff
@@ -0,0 +1,30 @@
+name: PHP Static Analysis
+on: [push, pull_request]
+jobs:
+  phpstan:
+    name: phpstan
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v1
+        with:
+          php-version: '7.3'
+          extensions: mbstring, intl, gd, xml, dom, json, fileinfo, curl, zip, iconv
+          tools: phpstan
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - run: composer install --prefer-dist
+
+      - name: "Create cache directory for phpstan/phpstan"
+        run: "mkdir -p .build/phpstan"
+
+      - name: "Cache cache directory for phpstan/phpstan"
```

Comment:
Duplicate name with above

Reply:
"Create cache directory for phpstan/phpstan" is not duplicate with "Cache cache directory for phpstan/phpstan"

Reply:
Hoho 😄 read too fast, did not see the 1st word was different

Tricky 😛

Reply:
"Cache cache" 😅